apps/api/src/app/workflows/e2e/get-workflow.e2e.ts

```diff
@@ -0,0 +1,32 @@
+import { expect } from 'chai';
+import { NotificationTemplateService, UserSession } from '@novu/testing';
```

Comment:
Not for this one but might need to change the DAL service name too in the near future for consistency. Unfortunately schema will might have to remain.